tests/prometheus/service.go

```diff
@@ -0,0 +1,48 @@
+// Copyright 2020 TiKV Project Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
```

Comment:
This license is short of a sentence, please use the latest one.

Reply:
short of what? I copied this from other file header, could you tell me a file path that I can copied from?